lib/alchemy/test_support/factories/page_factory.rb

```diff
@@ -21,13 +21,13 @@
       name 'Startseite'
       page_layout { language.page_layout }
       language_root true
-      public true
+      public_on { Time.current }
```

Comment:
Style/BlockDelimiters: Prefer `do...end` over `{...}` for procedural blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)